search-app/resources/templates/eosdis-landing-links.html

```diff
@@ -0,0 +1,9 @@
+{% extends "templates/base.html" %}
+
+{% block title %}CMR Search - Landing Pages for EOSDIS Collections{% endblock %}
```

Comment:
Updating routes to be `/site/collections/directory...` instead of using the misleading `landing-pages` page segment.